src/client/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -30,5 +30,6 @@ internal class AcquireTokenCommonParameters
         public Func<OnBeforeTokenRequestData, Task> OnBeforeTokenRequestHandler { get; internal set; }
         public X509Certificate2 MtlsCertificate { get; internal set; }
         public List<string> AdditionalCacheParameters { get; set; }
+        public SortedDictionary<string, string> CacheKeyComponents { get; internal set; }
```

Comment:
Why did you choose SortedDictionary over SortedList ? The docs say that SortedList is faster 

![image](https://github.com/user-attachments/assets/92f00cf1-ce05-4ac8-bf37-a857541ec522)

Reply:
- The spec uses a dictionary
- The the cache key components are a key value pair
So with that, a list didn't seem like the best data type to use here.

Reply:
Updated to sortedList